scratchpad/components/todo/controllers/TodoController.ts

```diff
@@ -0,0 +1,23 @@
+import {authenticate, authenticatedUser} from '@loopback/authentication';
+import {role} from '@loopback/authorization';
+import {todoRepo} from '../bindings/TodoRepo.ts';
+import {get} from '@loopback/core';
+
+export class TodoController {
+  constructor(
+    private @authenticatedUser() user,
+    private @todoRepo() todos
+  ) {}
+
+  @authenticate()
+  @get('/my/todos')
```

Comment:
FWIW, we don't support decorating methods with swagger metadata yet, we agreed the first step is to support external Swagger spec attached to a controller via `@api` decorator (which already works).

In order to keep our tasks smaller and more incremental, I am proposing to keep `@get('/my/todos')` outside of the scope of this work, and rework this controller to use `TodoController.spec.ts` and `@api`, as we are already doing in CodeHub.